gcloud/datastore/_generated/datastore_grpc_pb2.py

```diff
@@ -13,11 +7,19 @@
 from gcloud.datastore._generated.datastore_pb2 import CommitResponse
 from gcloud.datastore._generated.datastore_pb2 import LookupRequest
 from gcloud.datastore._generated.datastore_pb2 import LookupResponse
+from gcloud.datastore._generated.datastore_pb2 import Mutation
+from gcloud.datastore._generated.datastore_pb2 import MutationResult
+from gcloud.datastore._generated.datastore_pb2 import ReadOptions
 from gcloud.datastore._generated.datastore_pb2 import RollbackRequest
 from gcloud.datastore._generated.datastore_pb2 import RollbackResponse
 from gcloud.datastore._generated.datastore_pb2 import RunQueryRequest
 from gcloud.datastore._generated.datastore_pb2 import RunQueryResponse
-#   END: Manually added imports
+#   END: Imports from datastore_pb2
+import grpc
+from grpc.beta import implementations as beta_implementations
+from grpc.beta import interfaces as beta_interfaces
+from grpc.framework.common import cardinality
+from grpc.framework.interfaces.face import utilities as face_utilities
 
```

Comment:
Should imports from a third-party package like `grpc` be in a separate section above the "local" imports?